File: PROBLEM/1_2_D/CirinRec.c
#include<stdio.h>

int main(){
  int W,H,x,y,r,f=0;
  scanf("%d%d%d%d%d",&W,&H,&x,&y,&r);
  if(x+r>W) f=1;
  if(y+r>H) f=1;
  if(r>x) f=1;
  if(r>y) f=1;
  if(f==0) printf("Yes\n");
  else printf("No\n");
  return 0;
}

File: PROBLEM/1_2_D/CirinRec.py
l=input().split()
W=l[0]
H=l[1]
x=l[2]
y=l[3]
r=l[4]
f=0
if x+r>W:
    f=1
if y+r>H:
    f=1
if r>x:
    f=1
if r>y:
    f=1
if f==0:
    print("Yes")
else :
    print("No")
